scripts/setup_cli_env.sh

```diff
@@ -0,0 +1,77 @@
+0x03a34ef38f402d6b66b681db7905edfc48676288a7b08cd79910737c45431093:artifacts/abis/Food.json:proxy_Food
+0x0082593b45f86b7659f885604872e8ee63efea22d14fad6504a4dc2a4f74d8f1:artifacts/abis/L04_Calculator.json:L04_Calculator
+0x015eba242880374267dc54900b7d569a964fcd8d251a2edfb66a4ec9a78eaedc:artifacts/abis/Exchange_ERC20_1155.json:proxy_Exchange_ERC20_1155
+0x07220e609523f50a89a8070358a2d512c5c68dcca8afa5c41d7ac2406d8a5db8:artifacts/abis/Exchange_ERC20_1155.json:Exchange_ERC20_1155
+0x04dd3bb52964c9c57e8d1fd2144604a993cc5cb517182573a03b038666ab5705:artifacts/abis/SingleSidedStaking.json:SingleSidedStaking
+0x0259f9adda2c8a7e651d03472cb603ef2c69ae9a64fd3a553415d082ddbb3061:artifacts/abis/SingleSidedStaking.json:proxy_SingleSidedStaking
+0x020255b2f2079308299821c57437b17bd4a65183b3d4ac9b7532af9aebb976c0:artifacts/abis/Splitter.json:Splitter
+0x06a60b479e9fe080fd8e0a8c4965040a25e276889c2de0cf105c410d0ac81436:artifacts/abis/Splitter.json:proxy_Splitter
+0x04926f7b30bc33eb800e64992a669d4dcea96aa1fc9c1673cd9e1853487bc422:artifacts/abis/Relics.json:Relics
+0x03f233b0c1aa84b4a05ff55c7a3a0c1f6ba876030b06200ac27e80f338f59120:artifacts/abis/GoblinTown.json:proxy_GoblinTown
+0x02d8ae390c432eecc12f18d1c7c0e0283fee8abe6c4eecdae03753ec75cee3d8:artifacts/abis/L01_Settling.json:L01_Settling
+0x05983a41fad00287a3aa4dac3c385c1df7ba7af543fe3e7f3dfe4feb380b9fc4:artifacts/abis/L02_Resources.json:L02_Resources
+0x015586088996583d7dfa0cbe081aec0761275cd88be19eff8a61e652cca32bde:artifacts/abis/L03_Buildings.json:L03_Buildings
+0x07a0f46622f20fdd2bc1c212acb8412f66e48ee6b076b0bdf13f5640b61042f9:artifacts/abis/Food.json:Food
+0x0139de3bb4fc551b4c30f73256b822bb130208ad27d3cdbede7a8af706ef2ddf:artifacts/abis/GoblinTown.json:GoblinTown
+0x026ae695e80142822f5bbff3d58c5763d5de6cef1e67433ad5e5efd425ece250:artifacts/abis/Loot.json:proxy_Loot
+0x07c5ad28e6ce3aee36a72b065fa4bd8a2afa29a9dbd37e642f6965497e88c380:artifacts/abis/Adventurer.json:proxy_Adventurer
+0x02f850d76b3011d67ca4828785838d1ad4930124f7369e4983afe0ce9b638eae:artifacts/abis/Adventurer.json:Adventurer
+0x0426af93209f44ae0e5d27ab93b9180af32baa95502811df8ae6892d9874bfdf:artifacts/abis/Loot.json:Loot
+0x047024ede6449b6a980ca46faaa8180d48ca0b0f31fb8a42c51e45a9e84e0e57:artifacts/abis/L06_Combat.json:L06_Combat
+0x04d4e010850d0df3c6fd9672a72328514acc5e1285935104a29d215184903582:artifacts/abis/Combat.json:proxy_Combat
+0x05f273c4a45dab6e8112e2370bd84f58cfd2f1ff83752c2582241c0c0acba9be:artifacts/abis/Travel.json:proxy_Travel
+0x0045e9349f703c44e32f9851d5b9409f0d1bd55ca2f061a894a29397c9c1ea9a:artifacts/abis/Travel.json:Travel
+0x063431c98f0a5b9f4187b8c5616c8063d37e8e2d7b02ff9796c777bf5922205a:artifacts/abis/Combat.json:Combat
+$STARKNET_ACCOUNT_ADDRESS:/usr/local/lib/python3.9/site-packages/nile/artifacts/abis/Account.json:account-0
+EOT
+
+cat <<EOT >> goerli.accounts.json
+{"$STARKNET_PUBLIC_KEY": {"address": "$STARKNET_ACCOUNT_ADDRESS", "index": 0, "alias": "STARKNET_PRIVATE_KEY"}}
```

Comment:
this get's auto-created by nile same with the account. Perhaps cleaner if we let nile do it?

Reply:
@ponderingdemocritus It only gets created when we run `nile setup STARKNET_PRIVATE_KEY` though right? Which we wouldn't do if they are importing the PK and account from Argent?

Reply:
true, so let's leave it